src/validators/node-contacts.validator.js

```diff
@@ -26,12 +26,20 @@ const nodeSchema = {
 
           source_node_id: joi
             .string()
-            .guid()
+            .max(15)
             .required(),
 
-          node_pairs: joi
-            .array()
-            .items(joi.string().guid())
+          node_pair: joi
+            .string()
+            .max(15)
+            .required(),
+
+          rssi: joi
+            .number()
+            .required(),
+
+          txPower: joi
+            .number()
             .required(),
```

Comment:
Kindly revert the handling of array `node_pairs`. Also modify to validate the 15 character limit.

Reply:
Oops my bad. Backend na nga pala yung handling nung conversion. Oks na to.